src/libcore/future/future.rs

```diff
@@ -99,6 +99,18 @@ pub trait Future {
     fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output>;
 }
 
+/// Conversion into a `Future`.
+#[unstable(feature = "into_future", issue = "0")]
```

Comment:
We should file a tracking issue, but let's hold off just a bit.

Reply:
There should be a tracking issue before this gets merged, though, I think.